server/mocking.js

```diff
@@ -99,7 +99,7 @@ async function request(url, method = 'get', body = null, headers = {}) {
     body,
   })
   if (res.status >= 400) {
-    throw new Error(res.status)
+    // throw new Error(res.status) // TODO: mock on cypress level
```

Comment:
Could you please explain why this was changed?

Reply:
Initially, the testnet backend returned 500 on pool recommendation because it did not have any recommended pool set up. This place on the frontend therefore threw, I set up the backend to not throw. I'll revert this